Update tutorial-design-database-using-cli.md

tmichael-ms opened the pull request:
Fixed incorrect parameters to create MySQL server.  Also, made the Create MySQL server a single line of code so copy/paste works.

azurecla commented:
Hi __@bcraun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@bcraun Thanks for the contribution!

tmichael-ms committed:
Update tutorial-design-database-using-cli.md

Fixed incorrect parameters to create MySQL server.  Also, made the Create MySQL server a single line of code so copy/paste works.
